packages/card/src/css/index.js

```diff
@@ -140,6 +140,11 @@ export default {
     transition: `opacity ${core.motion.speedNormal}`,
     pointerEvents: 'none',
     opacity: 0,
+    zIndex: 10,
```

Comment:
I really like to avoid zIndex application where possible.  I feel like it's possible here.  One way to avoid it is to render the ActionMenu outside of the DOM of the card.  One could do that using React.createPortal.

Layering is just such a mess (or can be), I love to try avoid it.  

Do you feel like a portal (or other way to render the ActionMenu elsewhere in the DOM) is a good way to accomplish this?

Reply:
Hmm, we could probably figure out how to do it with portals but it seems like that would add a lot of complexity around positioning the menu in the right place. Not sure I'd want to take that on when a z-index works just as well. I definitely understand you wanting to avoid layering though.